openmetadata-spec/src/main/resources/json/schema/metadataIngestion/databaseServiceMetadataPipeline.json

```diff
@@ -93,6 +93,12 @@
       "description": "Regex to only fetch databases that matches the pattern.",
       "$ref": "../type/filterPattern.json#/definitions/filterPattern",
       "title": "Database Filter Pattern"
+    },
+    "threads": {
+      "description": "Number of Threads to use in order to paralellize Table ingestion.",
```

Comment:
suggestion: document how this translates to actual load on the data source (even a loose formula like 1 thread ~ 1 connection)